docker/vendors-sdk/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "dockerfiles"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "~3.10"
+cloakensdk = "*"
+confluent-kafka = "==1.7.0"
```

Comment:
please add a comment why this is locked. If it needs to be locked make sure to ignore it in the dependabot config also otherwise dependabot will update it anyway